README.md

```diff
@@ -46,6 +46,7 @@ SDK support with custom configuration for kafka stream, that is :
 * DialTimeout : Timeout duration during connecting to kafka broker (time.Duration)
 * ReadTimeout : Timeout duration during consume topic from kafka broker (time.Duration)
 * WriteTimeout : Timeout duration during publish event to kafka broker (time.Duration) 
+* LogMode : If it set true, eventstream will print log with debug level (boolean) 
```

Comment:
I think it's better to either:
- Use LogMode as switch between "off", "debug", "info", etc.
- Use DebugLog boolean

LogMode boolean is not intuitive

Reply:
I see, it makes more granular 
will update this 
thanks